{"uuid": "675b39d1-8ed1-4b9d-a34a-05dd40f12e76", "children": ["17cd6c40-1ca6-450d-b391-17b440252c63", "c7e909e3-161b-43fb-bf42-b312516ebd81", "f57f6747-2e46-4ced-885c-68b264f32f1a", "1457dcee-2294-4112-83b8-bc4a6e3911a0"], "befores": [{"name": "startApp", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'desired_caps.yaml'\n", "trace": "  File \"d:\\python\\python38\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"d:\\python\\python38\\lib\\site-packages\\_pytest\\fixtures.py\", line 1122, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"d:\\python\\python38\\lib\\site-packages\\_pytest\\fixtures.py\", line 917, in call_fixture_func\n    fixture_result = next(generator)\n  File \"D:\\JetBrains\\PyCharm\\PycharmProjects\\Duixi_Framework_V1\\TestCases\\Test0Welcome\\conftest.py\", line 33, in startApp\n    driver = baseDriver(noReset=False)\n  File \"D:\\JetBrains\\PyCharm\\PycharmProjects\\Duixi_Framework_V1\\TestCases\\Test0Welcome\\conftest.py\", line 49, in baseDriver\n    fs = open(\"desired_caps.yaml\")\n"}, "start": 1649164650535, "stop": 1649164650544}], "start": 1649164650535, "stop": 1649164650865}